numpy/core/setup_common.py

```diff
@@ -194,7 +194,7 @@ def check_api_version(apiversion, codegen_dir):
                                 ('__attribute__((target ("avx512f,avx512dq,avx512bw,avx512vl,avx512cd")))',
                                 'attribute_target_avx512_skx_with_intrinsics',
                                 '__mmask8 temp = _mm512_fpclass_pd_mask(_mm512_set1_pd(1.0), 0x01);\
-                                __m512i temp = _mm512_castps_si512(_mm512_set1_ps(1.0));\
+                                __m512i unused_temp = _mm512_castps_si512(_mm512_set1_ps(1.0));\
                                 _mm_mask_storeu_epi8(NULL, 0xFF, _mm_broadcastmb_epi64(temp))',
```

Comment:
Which `temp` is meant to be used in line 198?

Reply:
The one on line 196, it is using the correct input. I added an extra line of code in #16871 that used the same variable name which ended up disabling AVX512_SKX.